packages/pipcook-cli/lib/devPlugin.js

```diff
@@ -15,6 +17,11 @@ const devPlugin = (cmdObj) => {
     return;
   }
 
+  if (constants.PLUGINS.indexOf(pluginType) < 0) {
+    console.warn(chalk.red(`Unsupported plugin type: "${pluginType}", it must be one of: \n${constants.PLUGINS.join(',\n')}`));
```

Comment:
This line is too long, namely over 80 chars.

Reply:
Yeah, and many lines in this project are over 80 chars like this, we need a lint for standardized the code:)

Reply:
Ok, resolved.